### ISSUE TITLE
Allow user to access fastboot instance in fastboot-testing.js config

### ISSUE THREAD

**Issue by haochuan** [embermap/ember-cli-fastboot-testing#159]
## Feature Request

This is the follow up feature request for https://github.com/embermap/ember-cli-fastboot-testing/issues/57

Currently in `config/fastboot-testing.js`, we have a way to customize the sandbox provided to FastBoot:

```js
module.exports = {
  resilient: false,
  sandboxGlobals: {}
};
```

Add it would be super useful if in the config file, **we can have a way to access the fastboot instance**, which is created in the `_createServer` function inside `index.js`.

## Proposal

Let's say I have the following `config/fastboot-testing.js`:

```js
module.exports = {
  resilient: false,
  sandboxGlobals: {},
  setupFastboot: fastbootInstance => {
    // do something with the fastbootInstance here
  },
};
```
To make it work, we need to add the checking below in `index.js`:

```js
_createServer(distPath) {
   // ...
  this.fastboot = new FastBoot(fastbootOptions);

  // In test config, you can define a function `setupFastboot` to have access to the fastboot instance
  if (options.setupFastboot && typeof options.setupFastboot === 'function') {
    options.setupFastboot(this.fastboot);
  }
}
```

Please let me know if that makes sense, I will have a PR for this soon 😄 . Thanks!

**Comment by ryanto** [comment]
Hey @haochuan 

I think this sounds like a good idea!

Just a quick question first, what sort of things do you need this function for? I'd like to get an idea of the use case.

**Comment by haochuan** [comment]
@ryanto 

In our use case, we primarily need:

1. Assign `fastbootInstance._app.sandbox.sandbox` to `fastbootInstance._app.sandbox.sandbox.global`.

2. Have the access to the build `distPath`.

The PR is ready but I'm struggling about how to write this for this piece. Do you have any idea about it? Thanks!

**Comment by ryanto** [comment]
Ok sounds good...

> The PR is ready but I'm struggling about how to write this for this piece. Do you have any idea about it? Thanks!

Do you mean tests?

**Comment by haochuan** [comment]
Yes sorry for the typo

**Comment by ryanto** [comment]
Awesome! Thanks. This is released as version  `0.2.2`
